src/main/g8/default.properties

```diff
@@ -1,2 +1,2 @@
-name=Dotty Cross Compiled Project Template
-description=A template to demonstrate a minimal Dotty/Scala 2 cross compiled application
+name=Dotty Cross
```

Comment:
Why? As far as I can tell, this name only appears when you create the project:
```shell
sbt new lampepfl/dotty-cross.g8
A template to demonstrate a minimal Dotty/Scala 2 cross compiled 
application 

name [Dotty Cross Compiled Project Template]: 

Template applied in ./dotty-cross-compiled-project-template
```
The default name being long means that the project directory ends up with a long name (and somewhat confusing: this is not a template, this is an instance of a template)